TurkishFilter/sections/css_extended.txt

```diff
@@ -13,6 +13,7 @@ haberetanik.com#?#body > div.container > div.header_big_advert:upward(1)
 popkedi.com#?#div[data-testid="mesh-container-content"] > div[id^="comp-"] > iframe[aria-label="Google AdSense"]:upward(1)
 popkedi.com#?#div[data-testid="mesh-container-content"] > div[id^="comp-"] > wix-iframe > div > iframe[src^="https://www-popkedi-com.filesusr.com/html/"]:upward(3)
 320volt.com#?#.sidebar aside.widget a[rel^="sponsored"]:upward(1)
+memuralimlari.net#?#.row > div.col:has( > ins.rkm )
```

Comment:
Why did you use it? Extra spaces in `:has` part.
Also I don't see enough visitors for this site. Have you checked it?

Reply:
13k visitors, damn